src/Symfony/Component/Form/Extension/Core/Type/FileType.php

```diff
@@ -34,8 +34,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
             if ($options['multiple']) {
                 $data = array();
+                $files = $event->getData();
 
-                foreach ($event->getData() as $file) {
+                if (!is_array($files)) {
```

Comment:
What about checking for `null` here first and then dropping everything afterwards that is not an array?

Reply:
or maybe just:

```php
if (!is_array($files)) {
    $files = array(null);
}
```

Reply:
@xabbuh Thank you, I fixed it.